pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -119,6 +121,98 @@ func (configgen *ConfigGeneratorImpl) BuildClusters(proxy *model.Proxy, push *mo
 	return resources, model.XdsLogDetails{AdditionalInfo: fmt.Sprintf("cached:%v/%v", cacheStats.hits, cacheStats.hits+cacheStats.miss)}
 }
 
+// BuildDeltaClusters will generate the deltas (add and delete) for a given proxy. Currently, only service changes are reflected with deltas.
```

Comment:
nit: /s/BuildDeltaClusters will generate the deltas/BuildDeltaClusters generates